File: TransCoder_tokenized_test_set_functions/cpp/FIND_SUM_UNIQUE_SUB_ARRAY_SUM_GIVEN_ARRAY.cpp
long long int findSubarraySum ( int arr [ ], int n ) {
  int res = 0;
  unordered_map < int, int > m;
  for ( int i = 0;
  i < n;
  i ++ ) {
    int sum = 0;
    for ( int j = i;
    j < n;
    j ++ ) {
      sum += arr [ j ];
      m [ sum ] ++;
    }
  }
  for ( auto x : m ) if ( x . second == 1 ) res += x . first;
  return res;
}

File: results/java_codex/FIND_SUM_UNIQUE_SUB_ARRAY_SUM_GIVEN_ARRAY.java
int findSubarraySum(int arr[], int n) {
        int res = 0;
        unordered_map<int, int> m;
        for (int i = 0; i < n; i++) {
            int sum = 0;
            for (int j = i; j < n; j++) {
                sum += arr[j];
                if (m.find(sum) != m.end()) {
                    m[sum]++;
                } else {
                    m[sum] = 1;
                }
            }
        }
        for (auto x : m) {
            if (x.second == 1) {
                res += x.first;
            }
        }
        return res;
    }